### DISCUSSION TITLE
The R (programming language) Cask's CLIs/bins are symlinked under `/usr/local/bin` on ARM macs

### DISCUSSION THREAD

**Discussion by baggiponte** [Homebrew/discussions#3929] [Casks]
Hi! Installing R as a cask `brew install --cask r` results in `R` and `Rscript` binaries to be symlinked inside `/usr/local/bin`. This is fine on Intel Macs, not on ARM (where the `prefix` should be `/opt/homebrew/bin`. I had a look at the `r.rb` formula [here](https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/r.rb) and I am afraid I cannot understand where this behaviour could be modified. Can someone help?

**Comment by SMillerDev** [comment]
This is the way the R installer works. There's nothing Homebrew specific about those paths so it shouldn't write to `/opt/Homebrew`
